android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -342,7 +374,13 @@ public void setProgress(int max, int progress, boolean indeterminate)
 	public Notification buildNotification()
 	{
 		Notification notification = notificationBuilder.build();
-		notification.flags = this.flags;
+		
+		if (hasProperty(TiC.PROPERTY_GROUP_KEY)) {
+			// remove FLAG_AUTO_CANCEL as this will prevent group notifications
+			this.flags &= ~Notification.FLAG_AUTO_CANCEL;
+		}
+		notification.flags |= this.flags;
```

Comment:
Regarding `notification.flags |= this.flags`, is there a reason why you are changing it from an assignment to a bitwise-or operation? (Just curious in case there is something I'm not seeing here.)

I ask because the NotificationBuilder might set bits that can no longer be removed and the developer would have no means of clearing them. Versus the developer used to be able to set the exact flags he/she wanted before.